distributed/scheduler.py

```diff
@@ -626,9 +689,13 @@ def ensure_occupied_ready(self, worker):
 
         self._check_idle(worker)
 
+    def issaturated(self, worker, latency=5e-3):
```

Comment:
I think you should add a docstring to explain this heuristic:

- what this method it used for,
- what does "saturated" mean (and how this definition matches the code of the function).